src/lib/apis/fetch.ts

```diff
@@ -19,7 +19,8 @@ export const constructUrlAndParams = (method, url): URLAndRequestBodyParams => {
 
   if (method === "PUT" || method === "POST" || method === "DELETE") {
     const [path, queryParams] = url.split("?")
-    const parsedParams = parse(queryParams)
+    const parsedParams = parse(queryParams, { arrayLimit: 1000 })
```

Comment:
There is a useful option `throwOnLimitExceeded`, but I think it is not yet supported by our qs version

Reply:
Huh, interesting [that arrayLimit thing](https://github.com/ljharb/qs/blob/6cd60a59aae428123d45810996f4ac6b2c4e5c52/README.md?plain=1#L285-L287). I would not have anticipated that.